dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/message/MessageImpl.java

```diff
@@ -515,13 +725,12 @@ public void putAll(Map<? extends String, ? extends Object> m) {
     }
     @Override
     public Collection<Object> values() {
```

Comment:
Similar to the HashMap you can just have this be a instance variable that defaults to null and then make one and set it if it is null.  You don't need to make one every time it is called.